invenio_cli/helpers/scripts.py

```diff
@@ -122,28 +122,37 @@ def build_assets(local, statics, webpack, log_config):
         _build_container_assets(log_config, statics, webpack)
 
 
-def _force_symlink(file1, file2):
+def _force_symlink(target, link_name):
+    """Forcefully create symlink at link_name pointing to target."""
     try:
-        os.symlink(file1, file2)
+        os.symlink(target, link_name)
     except OSError as e:
         if e.errno == errno.EEXIST:
-            os.remove(file2)
-            os.symlink(file1, file2)
+            os.remove(link_name)
+            os.symlink(target, link_name)
 
 
 def update_config(local, log_config):
     """Update invenio.cfg configuration file."""
     if local:
-        # Create symbolic link
-        dst_path = _get_instance_path(log_config)
-        src_file = os.path.abspath('invenio.cfg')
-        _force_symlink(src_file, Path(dst_path) / 'invenio.cfg')
+        click.secho("Symlinking invenio.cfg...", fg="green")
+        target_path = os.path.abspath('invenio.cfg')
+        link_path = _get_instance_path(log_config) / 'invenio.cfg'
+        _force_symlink(target_path, link_path)
 
     else:
         # Copy to container
         pass
 
 
+def symlink_templates_folder(log_config):
+    """Symlink the templates folder (only used for local development)."""
+    click.secho("Symlinking templates folder...", fg="green")
+    target_path = os.path.abspath('templates')
+    link_path = Path(_get_instance_path(log_config)) / 'templates/'
```

Comment:
minor: Is the `Path()` needed? I see up in `updated_config` function:

``` python
        link_path = _get_instance_path(log_config) / 'invenio.cfg'
```

If it is needed, it will be in both cases, if so, shall if be part of `_get_instance_path` function?

Reply:
Thanks for catching this! `Path` is needed. I made `_get_instance_path` return a `Path` (it even formats correctly: `'{}/data'.format(Path('/home/user')) == '/home/user/data'`)